cves/kernel/CVE-2019-15666.yml

```diff
@@ -448,7 +486,8 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    This was lapse in judgement and did not take the array size into account.
```

Comment:
This could be reworded as "a lapse in judgement".

Reply:
I agree it's a lapse, but I think the out of bounds access is a red herring. The real issue is that `xfrm_policy_id2dir()` should be used to get the policy direction from the policy index. However, one place (the place in the patch) did not use that function but use a wrong method to do so, which gives a different result than using `xfrm_policy_id2dir()`